### ISSUE TITLE
Developers need to remove Command

### ISSUE THREAD

**Issue by zackeedoo** [TotalFreedom/TotalFreedomMod#2037]
The developers need to remove the command /msg ** because every time I log on there's a least one person who dose it and spams the chat. if you can you should put a 3 second wait on the /me command because people spam with that . From your fellow player zackeedoo.

**Comment by Wild1145** [comment]
This issue has been superseded by #436
